content/v2.md

```diff
@@ -193,6 +193,37 @@ were created in and notices based on these objects will be delivered with
 the proper locale. i.e. if an issue is created with the "es" locale, all
 email to the creator will be delivered with Spanish translations.
 
+## Terms of Use
+
+### API Rate Limits
+
+The rate limit allows for up to 20 requests per minute. 
+
+Requests are associated with the originating IP address.
+
+Rate limit is subject to change.
+
+#### HTTPS Headers and Response Codes
+
+If you go over the rate limits, we will return a HTTP 429 Too Many Requests error, and a Retry-After HTTP header containing the number of seconds until you can retry.
+
+```
+{
+    "message": "Too Many Requests",
+    "documentation_url": "http://dev.seeclickfix.com"
+}
+```
+
+```
+ Retry-After 10
+```
```

Comment:
Let's put a full syntactically accurate header/body example here, not just snippets

Reply:
I added a header

Reply:
What I meant was that you would show the information as a full HTTP response and not as separate header and body.  This is generated from httpstat.us but I was imagining the same thing from our service.
```
scfgrw:android$ curl -H "Accept: application/json" -i http://httpstat.us/429
HTTP/1.1 429 Too Many Requests
Cache-Control: private
Content-Length: 23
Content-Type: application/json; charset=utf-8
Retry-After: 3600
Server: Microsoft-IIS/10.0
X-AspNetMvc-Version: 5.1
Access-Control-Allow-Origin: *
X-AspNet-Version: 4.0.30319
X-Powered-By: ASP.NET
Set-Cookie: ARRAffinity=beec3692495883b8df6195e900c12f49514e054d865a22ad2951c84f51dbaf93;Path=/;HttpOnly;Domain=httpstat.us
Date: Fri, 01 Mar 2019 15:05:00 GMT

{
  "message": "Too Many Requests",
  "documentation_url": "http://dev.seeclickfix.com"
}
```

Reply:
Gotcha.  I haven't come across seeing it laid out that specifically but I can see the value.  I'll update.